pipeline/lib/cfn_nag/lib/cfn-nag/custom_rules/InternetGatewayRule.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require 'cfn-nag/custom_rules/base'
+
+class InternetGatewayRule < BaseRule
+  def rule_text
+    'Internet Gateways are not allowed'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W99'
+  end
+
+	def audit_impl(cfn_model)
```

Comment:
not a big deal, but well formatted code is typically easier to read

Reply:
Ok this should be fixed now, not sure why the format was funky lookin there lol